pkg/currency/service.go

```diff
@@ -55,3 +56,29 @@ func (service *CurrencyService) ToEur(value float64, from string) (float64, erro
 
 	return value / exchangeRate, nil
 }
+
+func (service *CurrencyService) ToCurrency(to string, value float64, from string) (float64, error) {
```

Comment:
Does it make sense to have a shorthand for `ToUsd`, as this is one of our most used currencies?